packages/commands/src/index.ts

```diff
@@ -1492,6 +1429,45 @@ namespace Private {
     event.target!.dispatchEvent(cloneKeyboardEvent(event));
   }
 
+  export function formatKey(key: string): string {
+    if (Platform.IS_MAC) {
+      return MAC_DISPLAY.hasOwnProperty(key) ? MAC_DISPLAY[key] : key;
+    } else {
+      return WIN_DISPLAY.hasOwnProperty(key) ? WIN_DISPLAY[key] : key;
+    }
+  }
+
+  const MAC_DISPLAY: { [key: string]: string } = {
+    Backspace: '⌫',
+    Tab: '⇥',
+    Enter: '↩',
```

Comment:
⏎ might have a closer semantic meaning, but will often render as a less familiar icon

Reply:
In VoiceOver, your character ⏎ is read as "return symbol", whereas the one in the PR (↩) is read as "return". I think the one in the PR is the official one to use.